ql/src/test/results/clientpositive/llap/lateral_view.q.out

```diff
@@ -100,22 +90,17 @@ STAGE PLANS:
               Limit
                 Number of rows: 1
                 Statistics: Num rows: 1 Data size: 178 Basic stats: COMPLETE Column stats: COMPLETE
-                Top N Key Operator
-                  sort order: ++
-                  keys: KEY.reducesinkkey0 (type: string), KEY.reducesinkkey1 (type: int)
-                  null sort order: zz
+                Select Operator
+                  expressions: KEY.reducesinkkey0 (type: string), VALUE._col0 (type: string), KEY.reducesinkkey1 (type: int)
+                  outputColumnNames: _col0, _col1, _col2
                   Statistics: Num rows: 1 Data size: 178 Basic stats: COMPLETE Column stats: COMPLETE
-                  top n: 1
-                  Select Operator
-                    expressions: KEY.reducesinkkey0 (type: string), VALUE._col0 (type: string), KEY.reducesinkkey1 (type: int)
-                    outputColumnNames: _col0, _col1, _col2
+                  Reduce Output Operator
+                    key expressions: _col0 (type: string), _col2 (type: int)
+                    null sort order: zz
+                    sort order: ++
                     Statistics: Num rows: 1 Data size: 178 Basic stats: COMPLETE Column stats: COMPLETE
-                    Reduce Output Operator
-                      key expressions: _col0 (type: string), _col2 (type: int)
-                      null sort order: zz
-                      sort order: ++
-                      Statistics: Num rows: 1 Data size: 178 Basic stats: COMPLETE Column stats: COMPLETE
-                      value expressions: _col1 (type: string)
+                    TopN Hash Memory Usage: 0.1
```

Comment:
When the plan contains a `Limit` operator all subsequent TopN optimizations in the same mapper/reducer seem redundant.